src/relayserver/ServerWinstonLogger.ts

```diff
@@ -0,0 +1,55 @@
+import winston, { Logger, transport } from 'winston'
+import { ConsoleTransportOptions, HttpTransportOptions } from 'winston/lib/winston/transports'
+import { URL } from 'url'
+
+import { gsnRuntimeVersion } from '../common/Version'
+
+import { NpmLogLevel } from '../relayclient/types/Aliases'
+
+const service = 'gsn-server'
+const filename = 'combined.log'
+
+const format = winston.format.combine(
+  winston.format.uncolorize(),
+  winston.format.timestamp(),
+  winston.format.simple()
+)
+
+const consoleOptions: ConsoleTransportOptions = {
+  format: winston.format.combine(
+    winston.format.cli()
+  )
+}
+
+export function createLogger (level: NpmLogLevel, loggerUrl: string, userId: string): Logger {
```

Comment:
please don't overload a method name exported by different places - especially when it gets different parameters...
this one is createServerLogger.

Reply:
Renamed